src/Goerli-DssSpell.sol

```diff
@@ -100,6 +102,10 @@ contract DssSpellAction is DssAction {
             100 * MILLION,
             1 weeks
         );
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
+            "RWA007-A",
+             500 * MILLION * WAD
+        );
```

Comment:
this is incomplete, please check [here](https://github.com/makerdao/dss-exec-lib/blob/master/src/DssExecLib.sol#L662) for a complete bump spell action

Reply:
Do we need some specific test for `spotter.poke()`?

Reply:
yes, improve coverage is always good, you can have a look [here](https://github.com/makerdao/spells-mainnet/blob/master/archive/2021-07-17-DssSpell/DssSpell.t.sol#L2285) also [here](https://github.com/makerdao/spells-mainnet/blob/master/archive/2021-07-17-DssSpell/DssSpell.sol#L56) for more context around the rwa oracle price bump spell action.

Reply:
done